### PR TITLE
feature/FOUR-10289: [story] Welcome Page

### PR DESCRIPTION
**Pull request by luNunezProcessmaker**
## Issue & Reproduction Steps
Fall-23-–-Figma (13)

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-10289

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.
ci:next
ci:deploy

### CODE REVIEW

File: resources/js/Mobile/WelcomeModal.vue
Comment (fagubla) [inline resources/js/Mobile/WelcomeModal.vue:1]: Add the new labels in en.json

File: resources/js/app-layout.js [hunk 1]
```diff
@@ -194,11 +196,17 @@ if (isMobileDevice) {
...
       requestModalMobile,
+      WelcomeModal,
     },
     data() {
       return {
       };
     },
+    mounted() {
+      if (this.$cookies.get("firstMounted") === "true") {
+        $("#welcomeModal").modal("show");
```
Comment (fagubla) [inline resources/js/app-layout.js:207]: this is like this:
https://bootstrap-vue.org/docs/components/modal#using-thisbvmodalshow-and-thisbvmodalhide-instance-methods

**Review by fagubla — CHANGES_REQUESTED** [review]

File: resources/js/app-layout.js (hunk 1 above)
Comment (luNunezProcessmaker) [reply resources/js/app-layout.js:207]: i don't know why, but with refs and b-modal it doesn't work

**Review by fagubla — APPROVED** [review]

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-9ede2d4256.eng.processmaker.net

**Pull request merged**